docs/content/component-data.md

```diff
@@ -9,56 +9,33 @@ SpatialOS components are generated from `.schema` files into components that the
 ### Overview
 
 A `struct`, which implements `Unity.Entities.IComponentData` and `Improbable.Gdk.Core.ISpatialComponentData`,
-is generated for each [blittable](https://docs.microsoft.com/en-us/dotnet/framework/interop/blittable-and-non-blittable-types) SpatialOS component.
-A `class`, which extends `UnityEngine.Component` and implements `Improbable.Gdk.Core.ISpatialComponentData` is generated for each non-blittable SpatialOS component.
+is generated for each SpatialOS component.
```

Comment:
Looking at this sentence and the one which precedes it in the doc, which I copy below...

"SpatialOS components are generated from .schema files into components that the Unity ECS can understand. See the schemalang docs for details on how to create schema components."

Neither the first sentence, or the one under `### Overview` tells us how the SpatialOS components are generated. What does the user have to do? What causes this to happen? It reads as if it happens by magic.

Reply:
Also:
I think the first sentence is an odd way round. It says "Spatial OS components are generated from .schema files into..."
I don't think anything can "be generated into" <a thing>. You can generate <a thing> and generate from <a thing>. We need to say what causes the generation,..
So:
"<A thing  - The Unity GDK?> uses `.schema` files to generate SpatialOS components into components that the Unity ECS can understand."

Reply:
I don't think explaining the code generation process is in scope for this doc - will link to the existing one. 

Code generation is (now) run automatically when you open Unity or when you press a button in the Editor - will make a note of that

Reply:
Just that addition above with and small introduction and when it happens is 100% clearer. <3 it.